module/TaskManagement/src/TaskManagement/Controller/RemindersController.php

```diff
@@ -73,6 +74,11 @@ public function create($data)
 		switch ($data['id']) {
 			case "assignment-of-shares":
 				
+				if(!$this->isAllowed($this->identity(), NULL, 'TaskManagement.Reminder.createReminder')){
```

Comment:
Il nome della risorsa deve riflettere la risorsa stessa, quindi in questo caso è meglio sia *TaskManagement.Reminder.assignment-of-shares*